azure-dataplane-tests/src/main/java/com/azure/ai/formrecognizer/documentanalysis/implementation/models/DocumentModelsAnalyzeDocumentHeaders.java

```diff
@@ -18,16 +18,14 @@ public final class DocumentModelsAnalyzeDocumentHeaders {
      */
     private String operationLocation;
 
-    private static final HttpHeaderName OPERATION_LOCATION = HttpHeaderName.fromString("Operation-Location");
-
     // HttpHeaders containing the raw property values.
     /**
      * Creates an instance of DocumentModelsAnalyzeDocumentHeaders class.
      * 
      * @param rawHeaders The raw HttpHeaders that will be used to create the property values.
      */
     public DocumentModelsAnalyzeDocumentHeaders(HttpHeaders rawHeaders) {
-        this.operationLocation = rawHeaders.getValue(OPERATION_LOCATION);
+        this.operationLocation = rawHeaders.getValue(HttpHeaderName.OPERATION_LOCATION);
```

Comment:
This change is likely due to azure-core change.

I remember the logic is that if `"Operation-Location"` is pre-defined in HttpHeaderName, it would use `HttpHeaderName.OPERATION_LOCATION`